ui/packages/catalog-ui-search/src/main/webapp/component/attributes-rearrange/attributes-rearrange.view.js

```diff
@@ -54,14 +54,7 @@ function calculateAvailableAttributesFromSelection(selectionInterface) {
       if (metacardDefinitions.metacardTypes[property]) {
         return !metacardDefinitions.metacardTypes[property].hidden
       } else {
-        announcement.announce({
-          title: 'Missing Attribute Definition',
-          message:
-            'Could not find information for ' +
-            property +
-            ' in definitions.  If this problem persists, contact your Administrator.',
-          type: 'warn',
-        })
+        console.warn('Could not find ' + property + ' in definitions.')
```

Comment:
❓This obviously makes the message no longer visible to the end users (unless they're running with the JS console). Is this intentional?

Reply:
yes. the message never worked to begin with, and I think it makes more sense as a console warning